test/lint/lint-circular-dependencies.py

```diff
@@ -22,6 +22,8 @@
     "wallet/wallet -> wallet/walletdb -> wallet/wallet",
     "node/coinstats -> validation -> node/coinstats",
     # Dash
+    "blockfilter -> evo/specialtx_filter -> evo/assetlocktx -> llmq/signing -> net_processing -> blockfilter",
```

Comment:
see c8ba1ae39eb85560a964fd4f72ac2477e1c41620

This same forward declaration is using in llmq/quorum, maybe it worth to create a new header for it: llmq/hash.h or llmq/io.h or something similar.

Or rename llmq/utils.h to llmq/quorum_utils.h and put BuildSignHash to lllmq/util.h